src/app/commands/cmd_timeline.cpp

```diff
@@ -28,7 +28,8 @@ class TimelineCommand : public Command {
 protected:
   void onLoadParams(const Params& params) override;
   void onExecute(Context* context) override;
-
+  bool onChecked(Context* ctx);
```

Comment:
Would you like to add "override" on this one? (like onLoadParams and onExecute). (PS. thanks for the patch :smile:)